e2e/test/framework/pod.go

```diff
@@ -91,3 +125,7 @@ func (f *Invocation) CheckIfFileIsInPod(filename string, pod *core.Pod) error {
 	}
 	return errors.Wrap(err, fmt.Sprintf("file name %v not found", filename))
 }
+
+func MkfsInPod(pod *core.Pod) error {
+	return runCommand("kubectl", "exec", "--kubeconfig", KubeConfigFile, "-it", "-n", pod.Namespace, pod.Name, "--", "/bin/bash", "-c", "mkfs.ext3 -F /dev/block")
```

Comment:
not sure it matters but it did say ext4 was the defaults... maybe this should be `mkfs.ext4` ?

Reply:
Sure.